Cargo.toml

```diff
@@ -14,7 +14,7 @@ exclude = ["/bors.toml", "/ci/*", "/.github/*"]
 edition = "2018"
 
 [package.metadata.docs.rs]
-features = ["bytemuck", "std", "serde", "rand"]
+features = ["bytemuck", "std", "serde", "rkyv", "bytecheck", "rand"]
```

Comment:
I suspect this will run into the same problem of needing a size choice. Maybe the same `"rkyv/size_64"` hack will work?

Reply:
The docs seem to build just fine regardless (probably because there aren't any rkyv docs examples). I will add the `size_64` anyway.